src/connection/sql_connection.go

```diff
@@ -70,6 +73,18 @@ func CreateConnectionURL(args *args.ArgumentList) string {
 	query.Add("dial timeout", args.Timeout)
 	query.Add("connection timeout", args.Timeout)
 
+	if args.ExtraConnectionURLArgs != "" {
+		extraArgsMap, err := url.ParseQuery(args.ExtraConnectionURLArgs)
+		if err == nil {
```

Comment:
just small sugestion for handling errors on Go, it common to check for != nil and handle the error below and if there is no error you can execute the code on the following lines.

Reply:
Thanks! This is my first golang contribution so I'm still learning. I appreciate the advice.